.github/workflows/ci_tests.yaml

```diff
@@ -42,8 +42,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9]
         os: [ubuntu-latest, macOS-latest, windows-latest]
+        # Run Python 3.6 tests on master branch only to conserve CI resources
+        isMaster:
+          - ${{ contains(github.ref, 'master') }}
+        exclude:
+          - isMaster: false  # i.e. on Pull Requests
+            python-version: 3.6
```

Comment:
It looks good to me, but perhaps we have to change the branch protection rules?

Reply:
Yep, we'll need to change the required checks from 3.6 to 3.9 (keeping 3.7 and 3.8 intact). Will do so when ready to merge.

Reply:
All CI jobs pass except the 3 that are excluded.
![image](https://user-images.githubusercontent.com/3974108/99141429-3dee5700-2619-11eb-84b3-46fc1c2ef066.png)

Reply:
Ok, branch protection rules updated!